example/go.mod

```
@@ -0,0 +1,24 @@
+module github.com/google/go-github/example/v43
```

I'm thinking that it might be better to mimic the other sub-modules and remove the version number here:

```bash
$ grep module */go.mod
scrape/go.mod:module github.com/google/go-github/scrape
update-urls/go.mod:module github.com/google/go-github/update-urls
```

```suggestion
module github.com/google/go-github/example
```

---

Done!